### DISCUSSION TITLE
stories_things/basic_content/decltype/

### DISCUSSION THREAD

**Discussion by giscus** [Light-City/light-comments#15] [Announcements]
# stories_things/basic_content/decltype/

Stories About C Plus Plus

https://light-city.github.io/stories_things/basic_content/decltype/

**Comment by cxz0218** [comment]
可以通过std::cout << boost::typeindex::type_id_with_cvr<decltype(var1)>().pretty_name() << std::endl; 输出类型检查

记得加上#include <boost/type_index.hpp>
